### PR TITLE
Functionality for: switching to 'Attack Mode'

### PR DESCRIPTION
**Pull request by aalokb**
Added modal for continuing to attack mode, as well as the code for showing and hiding elements when transitioning from place -> attack mode.

Note: if you're developing a modal or some attack functionality, I added comments to where you should show and hide your stuff, its all in the same block of code...

### CODE REVIEW

File: src/main/java/assets/game.js [hunk 1]
```diff
@@ -191,20 +190,27 @@ function initGame() {
         game = data;
     });
 
+    $('#move_to_attack_mode_button').click(function(){
```
Comment (kramerje) [inline src/main/java/assets/game.js]: Very clear and detailed with the comments, nice.

File: src/main/java/views/ApplicationController/index.ftl.html [hunk 2]
```diff
@@ -29,6 +29,15 @@ <h1>The Battleship game</h1>
 </div>
 
 </body>
+
+<div id="move_to_attack_mode_prompt">
```
Comment (kramerje) [inline src/main/java/views/ApplicationController/index.ftl.html]: I like this addition to our very early design ideas. Good, clear message to the user to let them know what is going on. Great job.

File: src/main/java/assets/css/custom.css [hunk 3]
```diff
@@ -93,4 +107,10 @@ td:hover {
 
 .miss {
     background-color: blue;
+}
+
+#ninja_footer {
```
Comment (kramerje) [inline src/main/java/assets/css/custom.css]: Good eye seeing this.

File: src/main/java/views/ApplicationController/index.ftl.html (hunk 2 above)
Comment (aalokb) [reply src/main/java/views/ApplicationController/index.ftl.html]: Yeah im currently thinking about the design of the modal, right now its pretty ugly.

**Pull request merged**